### DISCUSSION TITLE
Job and asset folders

### DISCUSSION THREAD

**Discussion by gnilrets** [dagster-io/dagster#17911] [Ideas]
I'm starting to have more jobs and asset groups and want to organize them further.  It would be cool if I could name an asset group something like "snowpark/finance" or "snowpark/ops" and then in the UI "finance" and "ops" would show up in an expandable folder.

**Comment by yuhan** [comment]
We are actively working on improvements to job and asset grouping. Relevant issues:
- https://github.com/dagster-io/dagster/issues/11826
- https://github.com/dagster-io/dagster/issues/6146
- https://github.com/dagster-io/dagster/issues/14182

cc @braunjj
